index.html

```diff
@@ -0,0 +1,1305 @@
+              </h5>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="row  pl-lg-4 pr-lg-4 bg-white">
+      <div class="col-12 mt-5">
+        <section class="last-articule">
+          <h5 class="last-articule-text mt-4">U.S.</h5>
+        </section>
+      </div>
+
+
+      <div class="col-lg-4 col-md-4 col-sm-12 col-xm-12 w-100 d-flex">
+        <div class="card w-100 ">
+          <img class="img-fluid w-100"
+            src="https://d.newsweek.com/en/full/1572864/seattle-washington-skyline-space-needle.jpg?w=397&amp;h=265&amp;f=e2e4a28646df0803522f405e721cdcf2"
+            alt="newsweek-img">
+          <div class="card-body pl-0 ">
+            <div class="card-title">
+              <h5 class="font-weight-bold">Easing of Coronavirus Quarantine Measures in Sacramento Leaves Experts Torn
+              </h5>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-lg-4 col-md-4 col-sm-12 col-xm-12 w-100">
+        <div class="card w-100 ">
+          <img class="img-fluid w-100"
+            src="https://d.newsweek.com/en/full/1572718/coronavirus-covid-19-statista.jpg?w=397&amp;h=265&amp;f=491f50e604dc16ba2b76c1991cba4d5d"
+            alt="newsweek-img">
+          <div class="card-body pl-0 ">
+            <div class="card-title">
+              <h5 class="font-weight-bold">Easing of Coronavirus Quarantine Measures in Sacramento Leaves Experts Torn
+              </h5>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-4 col-md-4 col-sm-12 col-xm-12 w-100">
+        <div class="media more-stories pb-3">
+          <img src="images/deck_img.jpg" class="mapping-small lazysize lazyloaded mr-3" alt="newsweek-img">
+          <div class="media-body">
+            <h5 class="media-title">More Aggressive' Coronavirus Strain May Have Evolved—But Don't Worry</h5>
+
+          </div>
+        </div>
+        <div class="media mt-3 more-stories pb-3">
+          <img src="images/deck_img.jpg" class="mapping-small lazysize lazyloaded mr-3" alt="newsweek-img">
+
+          <div class="media-body">
+            <h5 class="media-title">More Aggressive' Coronavirus Strain May Have Evolved—But Don't Worry</h5>
+
+          </div>
+        </div>
+        <div class="media mt-3 more-stories side  pb-3">
+          <img src="images/deck_img.jpg" class="mapping-small lazysize lazyloaded mr-3" alt="newsweek-img">
+
+          <div class="media-body">
+            <h5 class="media-title">More Aggressive' Coronavirus Strain May Have Evolved—But Don't Worry</h5>
+
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row  pl-lg-4 pr-lg-4 bg-white">
+      <div class="col-12 mt-5">
+        <div class="last-articule">
+          <h5 class="last-articule-text mt-4">WORLD</h5>
+        </div>
+      </div>
+
```

Comment:
The article images in all these sections (US, world, business, sports, culture...)  are wider due to the general width of your page. Please make sure that when you fix the body, this section reflects those changes, too.

In the medium breakpoint, the three articles of the side have larger images making the section look taller than the original one. Also, check the font size in this breakpoint and the small one.